cmd/notification.go

```diff
@@ -521,7 +521,7 @@ func sendEvent(args eventArgs) {
 		reqInfo.AppendTags("EventName", args.EventName.String())
 		reqInfo.AppendTags("targetID", targetID.Name)
 		ctx := logger.SetReqInfo(context.Background(), reqInfo)
-		logger.LogIf(ctx, err)
+		event.LogOnceIf(ctx, err, targetID)
```

Comment:
Can you move this function to logger itself? using logger outside of `cmd/` is not correct. Since other packages at pkg/event shouldn't be disturbed with code from cmd/*

Perhaps this can be generalized as well.

Reply:
It can be generalized by making the `id` an `interface{}` type